helm_deploy/whereabouts-api/values.yaml

```diff
@@ -1,18 +1,54 @@
 ---
 # Values here are the same across all environments
+generic-service:
 
-generic-prometheus-alerts:
-  targetApplication: whereabouts-api
+  image:
+    repository: quay.io/hmpps/whereabouts-api
+    tag: latest
+    port: 8080
+
+  ingress:
+    enabled: true
+    host: app-hostname.local
+    tlsSecretName: whereabouts-api-cert
+    contextColour: green
```

Comment:
This isn't required anymore and doesn't do anything - it can be removed from those projects it exists in.